src/main/scala/scalty/types/or.scala

```diff
@@ -113,15 +113,17 @@ object OrTypeExtensions {
   }
 
   final class OrExtensions[T](val value: T) {
-    def toOr: Or[T] = XorT.right[Future, AppError, T](Future.successful(value))(or.currentThreadExecutionFutureInstances)
+    def toOr: Or[T] = EitherT.fromEither[Future].apply(Right(value))(or.currentThreadExecutionFutureInstances)
 
     def toEmptyOr: EmptyOr =
-      XorT.right[Future, AppError, Empty](Future.successful(empty.EMPTY_INSTANCE))(or.currentThreadExecutionFutureInstances)
+      EitherT
+        .rightT[Future, AppError]
+        .apply[Empty](Future.successful(empty.EMPTY_INSTANCE))(or.currentThreadExecutionFutureInstances)
```

Comment:
This new object is independent of type `T`. Maybe it would be better to make it as constant object (extract it to `val`)?

Reply:
Or define `toEmptyOr` as `val`
```scala
@inline final val toEmptyOr: EmptyOr = ...
```